Documentation/requirements.md

```diff
@@ -0,0 +1,33 @@
+# Requirements
+
+Depending on the [platform](Platforms.md) that you are targeting, MonoGame has different sets of requirements.
+
+## For desktop platforms
+
+MonoGame requires a .Net Core SDK (3.1 or up) installation.
+
+You can either install it [independently](https://dotnet.microsoft.com/download/dotnet-core), or by selecting the .Net Core payload when installing Visual Studio 2019 (version 15.4 and up required).
+
+When it comes to IDE, Visual Studio 2019, Visual Studio Code, and Visual Studio 2019 for Mac are supported (alternatively, you can work directly from the CLI with your code editor of choice).
+
+Desktop development is possible from any operating system.
+
+## For UWP platforms
+
+MonoGame requires the latest Windows 10 SDK.
```

Comment:
I guess it does not really need to be the latest version, but this is definitely the easiest way to put it without going into Windows 10 SDK versioning and hardcoding a version here that would no doubt be outdated in the future because we forget to update it.

Reply:
Yeah, I guess it's safest to put it that way and may avoid us to edit the doc if any new SDK releases or if the requirements change.
The latest SDK is always shipped within VS 2019 and it allows to backtarget any Windows 10 version (and I believe that store submissions have to be using a quite recent SDK anyway).